Loyc-Slim.sln

```diff
@@ -0,0 +1,395 @@
+
```

Comment:
Right. That was my thinking, as well. And even if MonoDevelop/Xamarin Studio handles this case perfectly, there's still no telling how other IDEs may react to solution configurations that exclude projects. They kind of seem like an edge case to me, and I wouldn't be surprised if someone were to cut some corners when implementing them.

Ideally, we would just merge `Loyc.sln` and `Loyc-Slim.sln` in the future, by getting Baadia and LoycCore.Benchmarks to compile on Mono and AppVeyor. That'd be far more elegant than having either a separate `*.sln` or some build configuration magic, IMO. I'm just not sure how to do that right now, and I must say that it's not a top priority for me.